boards/xtensa/intel_adsp_ace15_mtpm/intel_adsp_ace15_mtpm_defconfig

```diff
@@ -12,3 +12,5 @@ CONFIG_GEN_IRQ_VECTOR_TABLE=n
 CONFIG_BUILD_OUTPUT_BIN=n
 CONFIG_MM_DRV=y
 CONFIG_CRYPTO=y
+
+CONFIG_DCACHE_LINE_SIZE=64
```

Comment:
There is one more Intel platform intel_adsp_ace20_lnl. Could you add the config to that one also?

Reply:
Sure, I wouldn't mind but this PR is only supposed to target boards used by SOF and as far as I can tell (please do correct me if I'm wrong) that one isn't one of them.

Reply:
The PR: https://github.com/thesofproject/sof/pull/7479 is still in review.

Reply:
ACK, will also do the change for that one.